packages/paste-core/components/button-group/stories/index.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import type {Story} from '@storybook/react';
+import {Button} from '@twilio-paste/core/button';
+import {ButtonGroup} from '../src';
+import type {ButtonGroupProps} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Button Group',
+  component: ButtonGroup,
+};
+
+export const Unattached: Story = (props: Partial<ButtonGroupProps>) => (
+  <ButtonGroup aria-label="A decisive group of buttons" {...props}>
+    <Button variant="secondary">Yes</Button>
+    <Button variant="secondary">No</Button>
+    <Button variant="secondary">Maybe</Button>
+  </ButtonGroup>
+);
+
+export const Attached: Story = (props: Partial<ButtonGroupProps>) => (
+  <ButtonGroup attached aria-label="A decisive group of buttons" {...props}>
+    <Button variant="secondary">Yes</Button>
+    <Button variant="secondary">No</Button>
+    <Button variant="secondary">Maybe</Button>
+  </ButtonGroup>
+);
```

Comment:
todo: need to add Customization stories

Reply:
could we also see a story with more or fewer buttons in the group than 3 if that's going to be allowed? if we're not restricting the number of button children, what might happen to a button group with a lot of button children in a small, fixed-width container? would it ever wrap?